packages/@react-stately/selection/src/useMultipleSelectionState.ts

```diff
@@ -52,6 +54,13 @@ export function useMultipleSelectionState(props: MultipleSelection): MultipleSel
       isFocusedRef.current = f;
       setFocused(f);
     },
+    get isFocusWithinCell() {
```

Comment:
Should we make the name more general than cell as it might apply to more than just tables?